packages/opentelemetry-node/test/NodeTracerProvider.test.ts

```diff
@@ -269,3 +278,32 @@ describe('NodeTracerProvider', () => {
     });
   });
 });
+
+describe('mergePlugins', () => {
+  const defaultPlugins = {
```

Comment:
Please add a test that if a user sets some config, the default config is not mangled.

```typescript
  const defaultPlugins = {
    module1: {
      enabled: true,
      path: 'testpath',
    },
    module2: {
      enabled: true,
      path: 'testpath2',
    },
  };

  const userPlugins = {
    module2: {
      path: 'userpath',
    },
  };

  const mergedPlugins = mergePlugins(defaultPlugins, userPlugins);

  // assert defaultPlugins.module2.path is testpath2
```